changelog/1.20.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: A site-wide workspace template policy may now be managed at **Manage >
+  Admin > Templates > Template Policy**. If not set, a sensible default is
+  provided.
+- web: The _nodeSelectors_ field was added to workspace templates
+- other: There is a new JsonSchema for writing Coder workspaces as code template
+  with intellisense
+- web: Added a service message that's displayed to all users of the system. The
+  message may be dismissed by each user at any point and is not shown again
+  until there is a new message.
+- web: System banners can now wrap text
+- infra: TLS management (/_TODO_/)
```

Comment:
Needs decision. If yes, needs documentation.

Reply:
I think we're delaying this until the next release, the APIs are available though... @lilshoff, thoughts?

Reply:
yes -delaying until next release